server/datastore/mysql/software_installers.go

```diff
@@ -1081,8 +1081,6 @@ WHERE
 
 func (ds *Datastore) GetSummaryHostSoftwareInstalls(ctx context.Context, installerID uint) (*fleet.SoftwareInstallerStatusSummary, error) {
 	var dest fleet.SoftwareInstallerStatusSummary
-	// TODO(uniq): do we need to support team filtering or is that not needed because installers are
-	// associated to teams?
```

Comment:
I removed this because it is as Sarah mentions in the comment (and not something we introduced in uniq work, it was already like that).